src/components/CromwellModal.js

```diff
@@ -0,0 +1,88 @@
+import _ from 'lodash/fp'
```

Comment:
Would love to see some UX done for this modal in the long term 😄

Reply:
Yes, that is the plan! That's why I put in the most bare bones, ugly thing I could. :)

Reply:
This also brings up an interesting question we haven't hit yet in terms of actually displaying the resources in the cluster the user configured for app#1 and allowing them to adjust when they create app#2.

cc @kyuksel.

Reply:
It'd be good to add a TODO in this file with a link to the ticket for refactoring.

Reply:
Sure, I can create a ticket for the creation of AppModal to share between GalaxyModal and CromwellModal. Though I'm still somewhat unsure about the best way to generalize all of this… I would really prefer that each app/tool isolate its code in one place, and the infrastructure code gets the things it needs through an established API. But that doesn't necessarily mean that Galaxy and Modal won't expose a modal class as part of that API (and their modal classes could use inheritance).

Reply:
@cahrens I think your thoughts would be a good discussion topic for the meeting @jdcanas scheduled 😊

Reply:
I put a TODO in the code instead of a JIRA ticket.

Reply:
I think it depends on the form the work takes. It may not make sense to create an AppModalBase class if Galaxy and Cromwell modals end up being fairly different.